### DISCUSSION TITLE
Concealing solutions that are excessively long

### DISCUSSION THREAD

**Discussion by kirillFedorov1** [codewars/codewars.com#3108] [Ideas]
Consider concealing solutions that are excessively long, such as those that span thousands of lines and are often submitted as "funny" solutions. These submissions can cause significant performance issues, freezing web browsers and creating a frustrating environment when attempting to view other, more serious solutions.

For instance, after completing the kata [You can't code under pressure #1](https://www.codewars.com/kata/53ee5429ba190077850011d4), I encountered an issue where my laptop was unable to load the solutions page at all.

**Comment by hobovsky** [comment]
I reported this as a bug in #3140 .
